src/policy.c

```diff
@@ -211,6 +213,8 @@ policy_set_sticky(int dev)
   new_rule->cmd = ALWAYS;
   new_rule->dev_vendorid = device->vendorid;
   new_rule->dev_deviceid = device->deviceid;
+  if (device->serial != NULL)
+    new_rule->dev_serial = device->serial;
```

Comment:
Added check to policy_set_sticky to ensure that if present, a serial is set for newly-created rules for auto-passing a device to a specified VM.